hub/core/index/index.py

```diff
@@ -263,12 +260,10 @@ def __init__(
         """
         if isinstance(item, Index):
             item = item.values
-        elif item in ((), [], None):
```

Comment:
Why is this removed?

Reply:
I don't know, after pulling some changes it gets removed automatically.